### ISSUE TITLE
row cover

### ISSUE THREAD

**Issue by jwping** [johnfercher/maroto#498]
At present, rows must have row height (when using 'AddAutorrow', it will also automatically calculate row height). In scenes where rows need to overlap with each other, if using top, left, right, or bottom, the height space occupied by the original rows will be left blank rows;
This scenario, for example: there is an image row that needs to be overlaid on top of another line (stamping scenario, where the image is an electronic seal)

May I ask if row can define a height that does not occupy the original space, or if there are other ways to achieve an image that can be overlaid on top of other rows (such as through top, left, right)

Looking forward to your clarification, thank you

**Comment by Fernando-hub527** [comment]
Hello !! 
I like help with this issue, but just to confirm that I understand your need: You just need the image to be overlaid on top of the text. Is that it?

**Comment by jwping** [comment]
Thank you very much for taking notice of my issue
Yes, for example, in the following situation:

![image](https://github.com/user-attachments/assets/924122c8-1d6c-49d7-bedd-a388c88c07e9)

The seal image needs to be placed on top of the text, and even the images can overlap with each other.
Thank you again for your attention

**Comment by Fernando-hub527** [comment]
No problem!
Got it!! It’s not exactly the same, but this issue feels a bit like issue #441. I think once Maroto supports watermarks, we’ll be able to implement this pretty easily.

**Comment by jwping** [comment]
Yes, they look similar. They should belong to the same category of problems, both of which require images to overlap on top of the content
Looking forward to your solution. Thank you for your contribution
